.gitignore

```diff
@@ -6,3 +6,4 @@ bin/rake
 .vagrant*
 test/tmp
 Gemfile.lock
+.idea
```

Comment:
I'd rather not put IDE-specific patterns in our gitignore. Those are more appropriate for a global gitignore (see <https://thomashunter.name/blog/global-gitignore-vs-repository-gitignore/>)